### PR TITLE
Added JCenter to samples

### PR DESCRIPTION
**Pull request by jbaruch**

### CODE REVIEW

File: subprojects/docs/src/docs/userguide/artifactDependenciesTutorial.xml [hunk 1]
```diff
@@ -157,7 +157,11 @@
         <sample id="defineMavenCentral" dir="userguide/artifacts/defineRepository" title="Usage of Maven central repository">
             <sourcefile file="build.gradle" snippet="maven-central"/>
         </sample>
-        <para>Or a remote Maven repository:</para>
+        <para>Or Bintray's JCenter:</para>
+        <sample id="defineJCenter" dir="userguide/artifacts/defineRepository" title="Usage of Jcenter repository">
```
Comment (eljobe) [inline subprojects/docs/src/docs/userguide/artifactDependenciesTutorial.xml]: Pick J**C**enter or J**c**enter and use the same capitalization on both lines 160 and 161.

Comment (jbaruch) [inline subprojects/docs/src/docs/userguide/artifactDependenciesTutorial.xml]: Fixed in #59f7581. Sorry.

**Comment by eljobe** [comment]
@jbaruch Thanks for making our documentation more complete. I don't have to tell you how important it is for the users to know about jcenter. :smile:

Thanks again,
Pepper

**Comment by jbaruch** [comment]
Cheers!

**Pull request merged**